plugins/modules/apache2_mod_proxy.py

```diff
@@ -246,7 +246,7 @@ class BalancerMember(object):
     """
 
     def __init__(self, management_url, balancer_url, module):
-        self.host = regexp_extraction(management_url, str(EXPRESSION), 4)
+        self.host = regexp_extraction(management_url, EXPRESSION, 4)
         self.management_url = str(management_url)
         self.protocol = regexp_extraction(management_url, EXPRESSION, 3)
         self.port = regexp_extraction(management_url, EXPRESSION, 5)
```

Comment:
This function is seaching for the same regex in the same string four times, every time using a different group of the result. That looks very inefficient to me... I guess this could be improved at some point :)